cookbooks/aws-parallelcluster-slurm/test/controls/config_health_check_spec.rb

```diff
@@ -52,7 +52,7 @@
 
   if instance.graphic?
     if instance.dcgmi_gpu_accel_supported?
-      if (os_properties.arm? && (os_properties.centos7? || os_properties.alinux2?)) || (os_properties.alinux2? && node['cluster']['nvidia']['enabled'] == 'no')
+      if (os_properties.arm? && (os_properties.centos7? || os_properties.alinux2?)) || (os_properties.alinux2? && (node['cluster']['nvidia']['enabled'] == 'no' || node['cluster']['nvidia']['enabled'] == false))
```

Comment:
Since 'no' and 'false' are semantically equivalent, why do we have different values and not just one (no or false)?